inst/qml/common/evaluation/EvaluationOutput.qml

```diff
@@ -107,6 +109,15 @@ Group
 			info:			qsTr("Produces a figure showing the range of plausible values for the misstatement for the population and all subpopulations (if applicable).")
 		}
 
+		CheckBox
+		{
+			text:			qsTr("Sequential analysis")
+			name:			"seqPlot"
+			visible: 		show_seqbf
+			enabled:		enable_seqbf	
+			info: 			qsTr("Produces a figure showing how the Bayes factor evolves throughout data collection.")	
```

Comment:
Maybe "Produces a figure showing how the Bayes factor evolves as a function of the sample size"?